src/styles/style.css

```diff
@@ -0,0 +1,159 @@
+.body {
+  margin: 0;
+
+  --border-color: rgb(240, 7, 228);
+
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 199px;
+  height: 100%;
+  border: 1px solid var(--border-color);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
```

Comment:
The `.card` block has a fixed width (199px), which is not recommended. Consider using max-width for better responsiveness or ensure this is intended as per design requirements.